db_operations/crud.py

```diff
@@ -0,0 +1,43 @@
+from sqlalchemy.orm import Session
+import db_operations.models
+import schemas

```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports)